### PR TITLE
📝 Generate IDL for emode debt cap release

### PR DESCRIPTION
**Pull request by andreihrs**
### Checklist

- [] if I'm working with the latest farms.so version, I have modified the so with the latest one from master
- [x] if I'm working with the latest kamino_lending.so version, I have modified the so with the latest one from master

### CODE REVIEW

File: src/classes/reserve.ts [hunk 1]
```diff
@@ -141,6 +141,10 @@ export class KaminoReserve {
     return new Fraction(this.state.config.fees.borrowFeeSf).toDecimal();
   };
 
+  getFixedHostInterestRate = (): Decimal => {
```
Comment (elliotkennedy) [inline src/classes/reserve.ts:144]:
```suggestion
  getFixedHostInterestRatePct = (): Decimal => {
```

Comment (elliotkennedy) [reply src/classes/reserve.ts:144]: Also add jsdoc

**Review by elliotkennedy — APPROVED** [review]
Looks good - just a comment around function naming to be more explicit - and it's always nice to add some docs to public interface methods if the meaning isn't immediately obvious or the client doesn't fetch the source code

File: src/classes/reserve.ts (hunk 1 above)
Comment (andreihrs) [reply src/classes/reserve.ts:144]: it's a fraction, I'll keep it as a rate. not a pct

**Pull request merged**